shared/actions/devices.js

```diff
@@ -32,6 +31,8 @@ const showRevokePage: (deviceID: string) => ShowRevokePage = deviceID => ({paylo
 
 const _loggedInSelector = (state: TypedState) => state.config.loggedIn
 
+const devicesTabLocation = isMobile ? [settingsTab, settingsDevicesTab] : [devicesTab]
```

Comment:
This is to help navigation on desktop/mobile be more explicit. We had a bug where if you add a computer and cancel you'd end up in the loginscreen cause the login saga would try and send you to [devicesTab] on mobile.